moveit_kinematics/test/fanuc.test

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<launch>
+	<group ns="fanuc">
+		<include file="$(find moveit_resources)/fanuc_moveit_config/launch/planning_context.launch">
+			<arg name="load_robot_description" value="true"/>
+		</include>
+
+		<param name="tip_link" value="tool0"/>
+		<param name="root_link" value="base_link"/>
+		<param name="group" value="manipulator"/>
+		<param name="manipulator/max_solver_iterations" value="50"/>
+		<rosparam param="joint_names">[joint_1, joint_2, joint_3, joint_4, joint_5, joint_6]</rosparam>
+
+		<!-- by default disable all tests: enable selectively with private parameters -->
+		<param name="num_fk_tests" value="0"/>
+		<param name="num_ik_tests" value="0"/>
+		<param name="num_ik_cb_tests" value="0"/>
+		<param name="num_ik_multiple_tests" value="0"/>
+		<param name="num_nearest_ik_tests" value="0"/>
+
+		<test test-name="fanuc_kdl" pkg="moveit_kinematics" type="test_kinematics_plugin" time-limit="180">
+			<param name="ik_plugin_name" value="kdl_kinematics_plugin/KDLKinematicsPlugin"/>
+			<!-- enable basic FK and IK tests -->
+			<param name="num_fk_tests" value="100"/>
+			<param name="num_ik_tests" value="100"/>
+			<!-- use a non-singular seed -->
+			<rosparam param="seed">[0, -0.32, -0.5, 0, -0.5, 0]</rosparam>
+			<rosparam param="consistency_limits">[0.4, 0.4, 0.4, 0.4, 0.4, 0.4]</rosparam>
+			<rosparam param="unit_tests">
```

Comment:
those are diffs applied individually, right? maybe the name could be more explicit ("unit_test_diffs"?) (and one of them be negative for demonstration purpose)
or maybe add a comment

Reply:
I added a comment.